src/VisualStudio/Core/Def/DocumentOutline/DocumentOutlineControl_ModelComputation.cs

```diff
@@ -45,24 +46,16 @@ private void EnqueueComputeDataModelTask()
             // further. We only get to the code below if the control is still in an active state.
             await _threadingContext.JoinableTaskFactory.SwitchToMainThreadAsync(cancellationToken);
 
-            var activeTextView = GetLastActiveIWpfTextView();
-            if (activeTextView is null)
-                return null;
-
-            var textBuffer = activeTextView.TextBuffer;
-            var filePath = GetFilePath();
-            if (filePath is null)
-                return null;
+            var textBuffer = GetLastActiveIWpfTextView()?.TextBuffer;
+            var filePath = textBuffer is null ? null : GetFilePath();
```

Comment:
instead of these checks here, you could just move the `if (textBuffer is null) return null;` into TryGetFilePath().  SImilarly, ComputeModuleAsync would jsut have to see if there was a null file path and could immediately return null.